src/main/java/com/fibermc/essentialcommands/teleportation/PlayerTeleporter.java

```diff
@@ -65,7 +66,8 @@ private static void execTeleport(ServerPlayerEntity playerEntity, MinecraftLocat
         playerEntity.teleport(
             targetWorld,
             dest.pos().x, dest.pos().y, dest.pos().z,
-            dest.headYaw(), dest.pitch()
+            PositionFlag.ROT, dest.headYaw(), dest.pitch(),
```

Comment:
Because we're setting the yaw and pitch explicitly from the `dest`, I don't think we want any relative positioning on ROT here. (I thing and empty set (`Set.of()`) would be what we want, probably?)

(Though this makes me realize an optional `pitch`/`headYaw` on `MinecraftLocation` may be useful, but that is a consideration for another time :P)